### PR TITLE
Multiregion

### PR DESCRIPTION
**Pull request by aintgoin2goa**
Adding a script to make the deployment to multiple buckets a bit easier (auto-replication too hard to set up in the FT)

### CODE REVIEW

File: deploy.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+'use strict';
+var nbt = require('next-build-tools');
+var path = require('path');
+
+var buckets = {
```
Comment (commuterjoy) [inline deploy.js]: buckets? 'environments' surely?

**Comment by commuterjoy** [comment]
> auto-replication

too hard because?

File: Makefile [hunk 2]
```diff
@@ -10,12 +14,20 @@ install:
 	npm install
 
 test:
-	mocha
 	nbt verify --skip-layout-checks
+	mocha
 
-deploy:
+publish:
```
Comment (matthew-andrews) [inline Makefile]: could we rename to build?

**Comment by matthew-andrews** [comment]
:+1:

**Pull request merged**